tools/github-event-processor/Azure.Sdk.Tools.GitHubEventProcessor/EventProcessing/IssueProcessing.cs

```diff
@@ -0,0 +1,489 @@
+        /// Conditions: Issue is open
+        ///             Label being added is "CXP Attention"
+        ///             Does not have "Service-Attention" label
+        /// Resulting Action: Add issue comment "Thank you for your feedback.  This has been routed to the support team for assistance."
+        /// Note: The comment added for this rule seems odd, since there's not much of anything of consequence in the comment.
+        /// The CXP team has a dashboard and automation specifically tied to the CXP Attention label. The comment is necessary to count
+        ///  as an initial response for SLA metrics.
+        /// </summary>
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <param name="issueEventPayload">IssueEventGitHubPayload deserialized from the json event payload</param>
+        public static void CXPAttention(GitHubEventClient gitHubEventClient, IssueEventGitHubPayload issueEventPayload)
+        {
+            if (gitHubEventClient.RulesConfiguration.RuleEnabled(RulesConstants.CXPAttention))
+            {
+                if (issueEventPayload.Action == ActionConstants.Labeled)
+                {
+
+                    if (issueEventPayload.Issue.State == ItemState.Open &&
+                    issueEventPayload.Label.Name.Equals(LabelConstants.CXPAttention) &&
+                    !LabelUtils.HasLabel(issueEventPayload.Issue.Labels, LabelConstants.ServiceAttention))
+                    {
+                        string issueComment = "Thank you for your feedback.  This has been routed to the support team for assistance.";
+                        gitHubEventClient.CreateComment(issueEventPayload.Repository.Id, issueEventPayload.Issue.Number, issueComment);
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Manual Triage After External Assignment
+        /// Trigger: issue unlabeled
+        /// Conditions: Issue is open
+        ///             Has "customer-reported" label
+        ///             Label removed is "Service Attention" OR "CXP Attention"
+        /// Resulting Action: Add "needs-team-triage" label
+        /// </summary>
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <param name="issueEventPayload">IssueEventGitHubPayload deserialized from the json event payload</param>
+        public static void ManualTriageAfterExternalAssignment(GitHubEventClient gitHubEventClient, IssueEventGitHubPayload issueEventPayload)
+        {
+            if (gitHubEventClient.RulesConfiguration.RuleEnabled(RulesConstants.ManualTriageAfterExternalAssignment))
+            {
+                if (issueEventPayload.Action == ActionConstants.Unlabeled)
+                {
+                    if (issueEventPayload.Issue.State == ItemState.Open &&
+                    LabelUtils.HasLabel(issueEventPayload.Issue.Labels, LabelConstants.CustomerReported) &&
```

Comment:
nit: some indention here might help understanding this condition.

Reply:
Good catch. It'll be updated with the next commit.